docs/quick-starts/bandit-quickstart.md

```diff
@@ -0,0 +1,200 @@
+```java
+// Flag that has a bandit variation
+String banditTestFlagKey = "bandit-test";
+
+// Subject information--same as for retrieving simple flag or experiment assignments
+String subjectKey = username;
+EppoAttributes subjectAttributes = userAttributes;
+
+// Action set for bandits
+Map<String, EppoAttributes> actionsWithAttributes = Map.of(
+  "dog", new EppoAttributes(Map.of(
+    "legs": EppoValue.of(4),
+    "size": "large"
+  )),
+  "cat", new EppoAttributes(Map.of(
+    "legs": EppoValue.of(4),
+    "size": "medium"
+  )),
+  "bird", new EppoAttributes(Map.of(
+    "legs": EppoValue.of(2),
+    "size": "medium"
+  )),
+  "goldfish", new EppoAttributes(Map.of(
+    "legs": EppoValue.of(0),
+    "size": "small" 
+  ))   
+);
+
+Optional<String> banditAssignment = eppoClient.getStringAssignment(subjectKey, flagKey, subjectAttributes, actionsWithAttributes);
+```
+
+If the actions don't have attributes for the context, you can simply provide a set of actions without attributes:
+```java
+Set<String> actions = Set.of("dog", "cat", "bird", "goldfish");
+
+Optional<String> banditAssignment = eppoClient.getStringAssignment(subjectKey, flagKey, subjectAttributes, actions);
+```
+
+Note that the `getAssignment` methods in Eppo are deterministic, meaning that they will always return the same variant for a given subject (e.g., user) for the life of the bandit.
+
+### 7. Train the Bandit 
+:::note
+Check that you're in the Test environment
+:::
+
+Using the template provided for Bandit Training, click `Add Feature Gate Allocation`. We are going to serve the Bandit to 100% of traffic in the test environment. Click the `Save` button to confirm.
+
+![Add bandit training to the Test environment](/img/bandit/bandit-qs-3.png)
+
+### 8. Turn the Bandit on in Test
+
+To start the Bandit, flip the flag on in the Test environment.
+
+![Turn on the bandit using the toggle](/img/bandit/bandit-qs-4.png)
+
+You should now see assignments coming through the Eppo SDK!
+
+### 9. Deploy to Production
+
+Complete these last steps to deploy your Bandit to Production:
+
+1. Create a new SDK key for the production environment from the SDK Keys tab
+2. In the Test environment, click the `Duplicate allocation` button next to Bandit Training and choose to copy it to Production
+![Copy the Bandit Training to Production](/img/bandit/bandit-qs-5.png)
+3. Switch to the Production environment using the environment switcher
+4. Using the template provided, click to add an experiment allocation for `Bandit Analysis`. This will allow you to measure the impact of your Bandit on a small percentage of traffic.
+![Production setup](/img/bandit/bandit-qs-6.png)
+5. Toggle the flag on in Production
+
+Congrats on running your first Bandit! As a next step, you can connect your flag to an Experiment Analysis so you can measure the impact your bandit is having on your product. See the [experiment quickstart guide](/experiment-quickstart) for more.
```

Comment:
I wonder if we should provide any color on what will happen. For example, until the bandit assignments are analyzed by our daily pipeline, an action will be randomly chosen. Each day, we'll analyze how assignments impacted the metric we're optimizing for, and start weighting action probabilities by the provided context and what we've learned.

Reply:
I think that's good in the context of the bandit reference doc, but not the quickstart doc